src/shared/util.js

```diff
@@ -1433,13 +1433,19 @@ MessageHandler.prototype = {
     let targetName = data.sourceName;
     let capability = createPromiseCapability();
 
-    let sendStreamRequest = ({ stream, chunk, success, reason, }) => {
+    let sendStreamRequest = ({ stream, chunk, transfers,
+                               success, reason, }) => {
+      if (transfers) {
```

Comment:
> and just let the postMessage method deal with it?

Yeah, right. Seems like `this.postMessage` is doing same thing, that I am trying here. Okay, calling just `this.postMessage` sounds like a good idea. Thanks.

> Also, don't you need to add similar code in _processStreamMessage below as well?

`sendStreamResponse` mostly used for signaling `*_complete` messages e.g. `pull_complete`(to resolve callbacks), and I don't think we are going to send _transfers_ in these types of messages, at least for now. Maybe we require this in future, but not sure.  @yurydelendik, do we require this?